test/functional/admin/publications_controller_test.rb

```diff
@@ -218,7 +218,7 @@ class Admin::PublicationsControllerTest < ActionController::TestCase
     get :show, params: { id: publication }
 
     assert_selected_specialist_sectors_are_displayed
-    assert_select "a[href='#{edit_admin_edition_legacy_associations_path(publication)}']", /Change Associations/
+    assert_select "a[href='#{edit_admin_edition_legacy_associations_path(publication)}']", /Change specialist topic tags/
```

Comment:
I'm just wondering whether it's worth renaming the controller and route for this... thoughts? https://github.com/alphagov/whitehall/blob/main/app/controllers/admin/edition_legacy_associations_controller.rb (plus template, test and references to it)

Reply:
And this features/support/admin_legacy_associations_helper.rb

Reply:
Yes I totally agree - there's a piece of work in all three apps following these label change PR's ([content tagger](https://github.com/alphagov/content-tagger/pull/1307), [collections-publisher](https://github.com/alphagov/collections-publisher/pull/1582))to sync the back end naming with the new frontend terminology. I thooooooought there was a card for that, but I can spot it in the backlog so I'll make one.